pkg/cluster-runtime/installer.go

```diff
@@ -35,7 +35,10 @@ import (
 	v1 "github.com/sealerio/sealer/types/api/v1"
 )
 
-var ForceDelete bool
+var (
+	ForceDelete bool
+	IsPrune     bool
```

Comment:
whether this parameter `IsPrune` should be put in the Distributor to do clean logic?

Reply:
Yep, PTAL